src/encoded/static/components/static-pages/components/StackedBlockVisual.js

```diff
@@ -245,30 +246,47 @@ export class StackedBlockVisual extends React.PureComponent {
         return out;
     }
 
-    static aggregateObjectFromList(dataList, keysToShow){
+    static aggregateObjectFromList(dataList, keysToShow, skipParsingKeys=null){
```

Comment:
The changes in this function are moot because decided on removing most of the properties from the popover anyway.
What changed is we skip parsing of categorization field (which is used as a title/key) into a JSX/React value, because we need to preserve it as string so it can work as an object key later in process. We also add if condition for handling number values as previously they would go into a list of numbers which then get uniqued into 1+ number that didn't really tell us anything useful (e.g. 'number_of_experiments' showing '2' for a block of 4 expsets that each have 2 number_of_experiments). Numbers now just get summed up from all expsets (if there were any displayed still).